packages/checkbox/README.md

```diff
@@ -43,3 +43,5 @@ import { Checkbox } from '@swc-uxp-wrappers/checkbox';
 ```html
 <sp-checkbox></sp-checkbox>
 ```
+## Known Issues
+- Tab navigation is not supported
```

Comment:
This is supported. Can you please confirm.

Reply:
This is dependant on opacity support for input widget in PS which is `in-progress`